src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Please, kindly use the duck's pattern as stated in the project requirements.  An example of duck's pattern:

```javascript
const ADD_BOOK = bookstore/books/ADD_BOOK
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

That way, we will meet this requirement 💪